database/__init__.py

```diff
@@ -1,4 +1,6 @@
-from .database import *
+from .database import get_cursor,\
+    insert_scan, insert_scan_grade, insert_test_result,\
+    select_scan_recent_scan, select_site_id, select_test_results
```

Comment:
Alternately, you could only import functions when and where you need them, rather than dumping in here. It makes the imports more complex, but it will save a bit on memory.

Tradeoffs `¯\_(ツ)_/¯`